dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
Is this state independent of submitted? If they're mutually exclusive and there's a chance we'll have more states, it make sense to combine them into an enum field.

Otherwise, if the point of this field is to restrict editing, then maybe something that makes that more clear like `readonly_answers`?